docs/README.md

```diff
@@ -86,7 +86,7 @@ description: Starship is the minimal, blazing fast, and extremely customizable p
 
    #### Powershell
 
-   Add the following to the end of `~\Documents\PowerShell\Microsoft.PowerShell_profile.ps1` (or `~/.config/powershell/Microsoft.PowerShell_profile.ps1` on -Nix):
+   Add the following to the end of `Microsoft.PowerShell_profile.ps1`. The location of this file can be checked with the `$PROFILE` command on PowerShell.
```

Comment:
I think it makes sense to mention both the file location and the `$PROFILE` variable. Maybe something akin to the [German translation](https://starship.rs/de-de/#powershell) of this?

Reply:
Thank you. I mentioned both `$PROFILE` variable and the location. My update is similar to German translation, I never noticed that before!